Makefile.include

```diff
@@ -287,6 +287,9 @@ term: $(filter flash, $(MAKECMDGOALS))
 		exit 1; }
 	$(TERMPROG) $(TERMFLAGS)
 
+list-ttys:
+	$(RIOTBASE)/dist/tools/usb-serial/list-ttys.sh
```

Comment:
Please prepend `$(AD)` to suppress output of the command:
`$(AD)$(RIOTBASE)/dist/tools/usb-serial/list-ttys.sh`